docs/WritingTests.asciidoc

```diff
@@ -477,13 +477,17 @@ sub run () {
 
 By default, any worker can get any job with the matching architecture.
 
-This behavior can be changed by setting job variable `WORKER_CLASS`. Jobs with
-this variable set (typically via machines or test suites configuration) are
-assigned only to workers, which have the same variable in the configuration
-file.
+This behavior can be changed by setting the job variable `WORKER_CLASS` taking
+a comma-separated list of worker class values. The values from multiple places
+of definition are combined. Typically machines and test suite configurations
```

Comment:
'of definition'? 
How about something like this instead: 
If the variable `WORKER_CLASS`, for a given job, is set in multiple places, the values are combined.

Is that what you try to say here? Sorry if I get this wrong; the sentence "The values from multiple places
of definition are combined." confuses me somehow.

Reply:
While it may be difficult to understand, "values from multiple places of definition" is a correct wording for what the sentence is trying to say.

Reply:
Hm, we could say "The values are combined from multiple places where defined". Would that be better?